client/cypress/e2e/_4layoutTopBar.cy.ts

```diff
@@ -0,0 +1,254 @@
+        cy.get('[data-test=MetricsView]').should('be.visible');
+        cy.get('[data-test=LayoutTopBar__link--home]').click();
+        cy.get('[data-test=HomeView]').should('be.visible');
+      });
+
+      it('Active/Inactive link has proper style', () => {
+        cy.get('[data-test=HomeView]').should('be.visible');
+        cy.get('[data-test=LayoutTopBar__link--home]')
+          .then($el => $el.css('color'))
+          .should('be.colored', '#171717');
+        cy.get('[data-test=LayoutTopBar__underline--home]').should('be.visible');
+        cy.get('[data-test=LayoutTopBar__link--projects]')
+          .then($el => $el.css('color'))
+          .should('be.colored', '#cdd1cd');
+        cy.get('[data-test=LayoutTopBar__link--metrics]')
+          .then($el => $el.css('color'))
+          .should('be.colored', '#cdd1cd');
+      });
+    }
+
+    it('Epoch info badge is visible', () => {
+      cy.get('[data-test=LayoutTopBarCalendar]').should('be.visible');
+    });
+
+    it('Epoch info badge opens Calendar on click', () => {
+      cy.get('[data-test=LayoutTopBarCalendar]').click();
+      cy.get('[data-test=Calendar]').should('be.visible');
+    });
+
+    it('Connect wallet button is visible', () => {
+      cy.get('[data-test=LayoutTopBar__Button]').should('be.visible');
+    });
+
+    it('Connect wallet button has proper label', () => {
+      cy.get('[data-test=LayoutTopBar__Button]')
+        .invoke('text')
+        .should('eq', device === 'mobile' ? 'Connect' : 'Connect wallet');
+    });
+
+    if (device === 'large-desktop' || device === 'desktop') {
+      it('Settings button is visible and has white background', () => {
+        cy.get('[data-test=LayoutTopBar__settingsButton]').should('be.visible');
+        cy.get('[data-test=LayoutTopBar__settingsButton]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#ffffff');
+      });
+
+      it('Settings button opens Settings drawer', () => {
+        cy.get('[data-test=LayoutTopBar__settingsButton]').click();
+        cy.get('[data-test=SettingsDrawer]').should('be.visible');
+      });
+
+      it('Allocation button is visible and has white background', () => {
+        cy.get('[data-test=LayoutTopBar__allocationButton]').should('be.visible');
+        cy.get('[data-test=LayoutTopBar__allocationButton]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#ffffff');
+      });
+
+      it('Allocation button opens Allocation drawer', () => {
+        cy.get('[data-test=LayoutTopBar__allocationButton]').click();
+        cy.get('[data-test=AllocationDrawer]').should('be.visible');
+      });
+    }
+
+    describe('[TESTNET]', () => {
+      beforeEach(() => {
+        cy.window().then(win => {
+          // eslint-disable-next-line no-param-reassign
+          win.isTestnetCypress = true;
+          // wait for re-render
+          cy.wait(2500);
```

Comment:
Maybe we can wait for something more specific? Force relaod and wait for apploader to disappear?

Reply:
After reload `win.isTestnetCypress` will be undefined (it's not a value set from LS -> only for tests)